### ISSUE TITLE
Add accessibility labels

### ISSUE THREAD

**Issue by HichamELBSI** [HichamELBSI/react-native-body-highlighter#77]
### Why

Each part should be accessible to screen readers. 

### What to do

- Add accessibilityLabel per body part.
- Document accessibility usage.

**Comment by HichamELBSI** [comment]
Hi @jjs2099 thanks for you contribution. The task is assigned to you.
